src/code/z_debug.c

```diff
@@ -33,8 +33,7 @@ InputCombo inputCombos[REG_GROUPS] = {
 
 char regChar[] = " SOPQMYDUIZCNKXcsiWAVHGmnBdkb";
 
-// initialize GameInfo
-void func_800636C0(void) {
+void GameInfo_Init(void) {
```

Comment:
Not sure about the `GameInfo` prefix

The way functions are named is `System_Name`

This would introduce the `GameInfo` system then, but that would be the only function in it.
We would need to know whether other functions would fit inside that system, to avoid this prefix getting a single use

Reply:
its probably ok, but would honestly prefer this be handled with the whole z_debug.c being handled at once to see if its appropriate. if youre not going to do that id leave it unnamed for now

Reply:
I may take a look at that later, but think I'll revert for now for simplicity sake.